src/class.rs

```diff
@@ -44,7 +44,7 @@ pub trait UsbClass<B: UsbBus> {
     /// * `index` - A string index allocated earlier with
     ///   [`UsbAllocator`](crate::bus::UsbBusAllocator).
     /// * `lang_id` - The language ID for the string to retrieve.
-    fn get_string(&self, index: StringIndex, lang_id: LangID) -> Option<&str> {
+    fn get_string(&self, index: StringIndex, lang_id: Option<LangID>) -> Option<&str> {
```

Comment:
I'm very hesitant about introducing another API change in the traits. It results in a ton of downstream work for various crates to re-release and is still not done for the 0.3.0 version.

Reply:
yeah I get it, I can go the simpler route and default to EN_US if that is ok.

Reply:
Lets go with that for now.